lib/core/include/rodsErrorTable.h

```diff
@@ -784,6 +784,7 @@ NEW_ERROR(INVALID_KVP_STRING,                   -1826000)
 NEW_ERROR(PLUGIN_ERROR_MISSING_SHARED_OBJECT,   -1827000)
 NEW_ERROR(RULE_ENGINE_ERROR,                    -1828000)
 NEW_ERROR(REBALANCE_ALREADY_ACTIVE_ON_RESOURCE, -1829000)
+NEW_ERROR(INVALID_RESOURCE_HIERARCHY,           -1830000)
```

Comment:
FWIW, `HIERARCHY_ERROR` is an existing error which may be equally effective. Otherwise, I would argue this should have a more specific name to inform the user as to what happened. Although, I suppose the client error stack will contain this information...

Reply:
i think i agree, HIERARCHY_ERROR should already communicate enough, along with the specific error message being sent back.